src/host/ledger.h

```diff
@@ -366,7 +366,11 @@ namespace asynchost
     {
       if ((from < start_idx) || (to > get_last_idx()) || (to < from))
       {
-        LOG_FAIL_FMT("Unknown entries range: {} - {}", from, to);
+        LOG_FAIL_FMT(
+          "Unknown entries range: {} - {} in ledger file {}",
```

Comment:
Maybe "Could not find entries {} to {} ..." is a little clearer, or maybe it's just me, I don't know :)